provider/datagen/src/transform/cldr/units/helpers.rs

```diff
@@ -139,18 +138,18 @@ pub(in crate::provider) fn process_factor(
     Ok(result)
 }
 
-/// Converts a vector of strings to a vector of string slices.
-fn to_str_vec(string_vec: &[String]) -> Vec<&str> {
-    string_vec.iter().map(|s| s.as_str()).collect()
-}
-
 /// Extracts the conversion info from a base unit, factor and offset.
 pub(in crate::provider) fn extract_conversion_info<'data>(
     base_unit: &str,
     factor: &ScientificNumber,
     offset: &ScientificNumber,
     parser: &MeasureUnitParser,
 ) -> Result<ConversionInfo<'data>, DataError> {
+    /// Converts a vector of strings to a vector of string slices.
+    fn to_str_vec(string_vec: &[String]) -> Vec<&str> {
```

Comment:
you don't need this if you change `convert_slices_to_fraction` to accept `&[String]`

Reply:
good idea